components/feature/intent/src/main/java/mozilla/components/feature/intent/processing/TabIntentProcessor.kt

```diff
@@ -85,7 +120,9 @@ class TabIntentProcessor(
         val safeIntent = SafeIntent(intent)
         return safeIntent.action == ACTION_VIEW ||
             safeIntent.action == ACTION_SEND ||
-            safeIntent.action == ACTION_NDEF_DISCOVERED
+            safeIntent.action == ACTION_NDEF_DISCOVERED ||
+                safeIntent.action == ACTION_SEARCH ||
```

Comment:
nit: could you fix the indentation to match the previous line? 🙂

Reply:
It should be addressed [here](https://github.com/mozilla-mobile/android-components/pull/5426/commits/45f2185308fa19165e91b0775ab78c74f915789e) :)

Reply:
@salpiras Thanks! Squashing those changes into your last commit would also be nice so that the commit log is easier to read. :)

Reply:
Done! :)